### PR TITLE
feat: [#188163509] add social media and general SEO to starter kits pages

### PR DESCRIPTION
**Pull request by adidner**
## Description

We wanted to add a social media preview image to the website for all major websites. 

We realized that we should in addition to this likely add some other SEO characteristics

### Ticket

This pull request resolves [#188163509](https://www.pivotaltracker.com/story/show/188163509).

### Steps to Test

If you go into my code and swap out the image URL base with http://localhost:3000/img/team-success.jpg,
Then download this https://chromewebstore.google.com/detail/social-share-preview/ggnikicjfklimmffbkhknndafpdlabib
Then activate it on a starter kits page you can see how the preview should look. 

Beyond that you could inspect and go into the <head> and look at the attributes (while on a starterkits page) and see that the <meta> were what you expected them to be. 

## Code author checklist

- [X] I have rebased this branch from the latest main branch
- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation on the engineering documentation website
- [X] I have not used any relative imports
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] I have checked for and removed instances of unused config from CMS
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [X] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/public/mgmt/config.yml [hunk 1]
```diff
@@ -3757,6 +3757,18 @@ collections:
                     name: "ctaButton"
                     widget: string
                     required: true
+              - label: "Seo"
+                name: "seo"
+                widget: object
```
Comment (AnJuHyppolite) [inline web/public/mgmt/config.yml:3762]: Is the `required` key not needed for the `SEO` label?

File: web/src/pages/_app.tsx [hunk 2]
```diff
@@ -122,10 +124,49 @@ const App = ({ Component, pageProps }: AppProps): ReactElement => {
...
+
   return (
     <>
       <Head>
         <meta name="viewport" content="width=360, initial-scale=1" />
+        {isSeoPage && (
+          <>
+            <meta property="og:title" content={heroTitle} />
+            <meta property="og:description" content={description} />
+            <meta property="og:image" content="https://navigator.business.nj.gov/img/team-success.jpg" />
```
Comment (seidior) [inline web/src/pages/_app.tsx:153]: [pebble] Do we not want to use `NEXT_PUBLIC_WEB_BASE_URL` here?

**Review by seidior — APPROVED** [review]
Worked on this with @adidner so I don't have much additional to contribute here atop Aaron's good work on research. Thanks again for handling this with the changing requirements!

File: web/public/mgmt/config.yml (hunk 1 above)
Comment (jphechter) [reply web/public/mgmt/config.yml:3762]: It's not. This is an object comprised of multiple fields. The individual fields can be required, but the overall container doesn't have an associated input.

Here's an example of what something similar looks like:
Screenshot 2024-09-09 at 3 04 05 PM

File: web/src/pages/_app.tsx (hunk 2 above)
Comment (jphechter) [reply web/src/pages/_app.tsx:153]: I think we'd want that for all of these instances, correct?

**Pull request merged**